modules/library/src/main/scala/zio/elasticsearch/ElasticQuery.scala

```diff
@@ -25,6 +25,36 @@ import zio.schema.Schema
 
 object ElasticQuery {
 
+  /**
+   * Constructs a type-safe instance of [[zio.elasticsearch.query.ConstantScoreQuery]] with a specified query.
+   * [[zio.elasticsearch.query.ConstantScoreQuery]] wraps a filter query and returns every matching document with a
+   * relevance score equal to the boost parameter value.
+   *
+   * @param query
+   *   query you wish to run
```

Comment:
Maybe something like this:

query to be wrapped inside of constant score query? Or something similar.